src/content-handlers/iiif/extensions/uv-openseadragon-extension/ISettings.ts

```diff
@@ -6,4 +6,5 @@ interface ISettings {
   preserveViewport?: boolean;
   clickToZoomEnabled?: boolean;
   reducedAnimation?: boolean;
+  extendThumbnailLabels?: boolean;
```

Comment:
As discussed on the February 1 Community Call: can you please reverse the naming/labeling here to make this "truncateThumbnailLabels" defaulting to true, instead of "extendThumbnailLabels" defaulting to false, in order to improve clarity?